### ISSUE TITLE
protoc-gen-go_cli: issue with generated oneof field

### ISSUE THREAD

**Issue by tmc** [googleapis/gapic-generator-go#536]
This codebase doesn't compile because of an apparent bug in the gen go_cli codebase:

```
cmd/secretsctl/create-secret.go:29:49: undefined: timestamp.Secret_ExpireTime
cmd/secretsctl/create-secret.go:31:42: undefined: duration.Secret_Ttl
cmd/secretsctl/update-secret.go:31:49: undefined: timestamp.Secret_ExpireTime
cmd/secretsctl/update-secret.go:33:42: undefined: duration.Secret_Ttl
```

The proto code in question is:
https://github.com/googleapis/googleapis/blob/master/google/cloud/secretmanager/v1/resources.proto#L81

https://github.com/tmc/secretsctl/issues/2

**Comment by noahdietz** [comment]
I tested the fix in #537 and it seemed to work. Also tested against our own Showcase API and a couple other Google APIs. Thanks for filing this.

**Comment by noahdietz** [comment]
Fix released in v0.18.4.
